core/src/components/toast/toast.tsx

```diff
@@ -138,6 +138,10 @@ export class Toast implements ComponentInterface, OverlayInterface {
    */
   @Method()
   async present(): Promise<void> {
+    if (this.showCloseButton) {
+      console.error(`showCloseButton has been deprecated, use "buttons" instead`);
```

Comment:
maybe `console.warn` would be more appropriate?

Reply:
The only other deprecation notice I saw in the ionic repo used an error as well. I can use warn though if that’s preferable